settings/provisioner/openstack/neutron/default.yml

```diff
@@ -0,0 +1,38 @@
+---
+# This is the most common network topology for OpenStack Testing deployments
+
+networks:
+    data:
+        name: '{{ random.uuid }}-data'
+    external:
+        name: '{{ random.uuid }}-external'
+    management: 
+        name: '{{ random.uuid }}-management'
+
+subnets:
+    data:
+        name: "{{ !lookup provisioner.neutron.networks.data.name }}"
+        network_name: "{{ !lookup provisioner.neutron.networks.data.name }}"
+        allocation_pool_start: 172.16.1.10
+        allocation_pool_end: 172.16.1.100
+        cidr: 172.16.1.0/24
+    external:
+        name: "{{ !lookup provisioner.neutron.networks.external.name }}"
+        network_name: "{{ !lookup provisioner.neutron.networks.external.name }}"
+        allocation_pool_start: 172.31.0.10
+        allocation_pool_end: 172.31.0.100
+        cidr: 172.31.0.0/23
+    management:
+        name: "{{ !lookup provisioner.neutron.networks.management.name }}"
+        network_name: "{{ !lookup provisioner.neutron.networks.management.name }}"
+        allocation_pool_start: 192.168.1.10
+        allocation_pool_end: 192.168.1.100
+        cidr: 192.168.1.0/24
+
+routers:
+    router:
+        name: '{{ random.uuid }}-router'
+        attach_subnets:
+            - "{{ !lookup provisioner.neutron.networks.external.name }}"
+            - "{{ !lookup provisioner.neutron.networks.management.name }}"
+        network: '10.8.184.0/22'
```

Comment:
:-1: 
Major -1

This shouldn't be hard coded.
We've discussed a solution for finding available public networks. But it must have some default. possible:

- public
- nova
- omit